### ISSUE TITLE
Error when using -vad_v3.1

### ISSUE THREAD

**Issue by dgoryeo** [linto-ai/whisper-timestamped#147]
I get this runtime error when running in Win11:

```
line 1873, in apply_folder_hack
    os.symlink(repo_or_dir_specific, repo_or_dir_master)
OSError: [WinError 1314] A required privilege is not held by the client: 'C:\\Users\\U/.cache/torch/hub/snakers4_silero-vad_v3.1' -> 'C:\\Users\\U/.cache/torch/hub/snakers4_silero-vad_master'
```

I'm running in conda powershell but not admin priviledges. 

Thanks

**Comment by Jeronymous** [comment]
Yes, in your case, you cannot use "vad=silero:3.1", unless you can change permissions of ''C:\\Users\\U/.cache/torch/hub" so that non admin user can write in it.
There is a bug in silero around version control, and the unique workaround I found was to mesh with cache folders.
If you are disappointed with the latest version of silero, you can use auditok and it should work smoothly.

(Note: please provide the input code / command you run when you report an error, so that everybody can understand the context of the error)
